pkg/util/error.go

```diff
@@ -40,3 +40,16 @@ func CheckErr(err error, handleErr func(string)) {
 	}
 	handleErr(err.Error())
 }
+
+// Handle Empty Table Error
+func HandleEmptyTableError(resource string, ns string, casType string) error {
+	if ns == "" && casType == "" {
+		return fmt.Errorf("no %s found in your cluster", resource)
+	} else if ns != "" && casType != "" {
+		return fmt.Errorf("no %s %s found in %s namespace", casType, resource, ns)
+	} else if casType != "" {
```

Comment:
Abhinandan has recently written a nice utility function which returns true for valid casTypes. Try to rebase your code against the current `origin/develop` and you'd likely see that in this package. Use that one for this condition check.

Reply:
Done